project-koku/koku
# Cost 2485 install pg stat statements

Fixes [COST-2485](https://issues.redhat.com/browse/COST-2485)

Changes proposed in this PR:
* Install the `pg_stat_statements` extension if it does not exist
* Remove the `pg_stat_statements` extension on unapply migration

Testing Instructions:
1. Checkout this branch
2. `make run-migrations` _(you should see 0055_install_pg_stat_statements)_
3. Launch `psql`
4. execute `\dx` -- _you should see pg_stat_statements in the data output_
5. exit `psql` with `\q`
6. `make run-migrations applabel=api migration=0054`
7. Repeat steps 3, 4. You should **not** see pg_stat_statements in the data output
8. Repeat steps 2 - 5. Step 4 should yield the same results.

---
Additional Context:
